core/dbt/graph/selector.py

```diff
@@ -62,8 +62,11 @@ def select_included(
         """Select the explicitly included nodes, using the given spec. Return
         the selected set of unique IDs.
         """
-        method = self.get_method(spec.method, spec.method_arguments)
-        return set(method.search(included_nodes, spec.value))
+        if spec.method == 'selector':
+            return self.get_selected(self.project.get_selector(spec.value))
+        else:
+            method = self.get_method(spec.method, spec.method_arguments)
+            return set(method.search(included_nodes, spec.value))
 
```

Comment:
The self.project.get_selector is the only place that checks if selectors actually exist